templates/Gemfile.erb

```diff
@@ -27,9 +27,10 @@ gem "fluent-plugin-rewrite-tag-filter", "~> 2.2.0"
 gem "elasticsearch", "~> 6.0"
 gem "fluent-plugin-elasticsearch", "~> 3.5.5"
 <% when "elasticsearch7" %>
-gem "elasticsearch", "~> 7.0"
-gem "fluent-plugin-elasticsearch", "~> 3.7.1"
+gem "fluent-plugin-dedot_filter", "~> 1.0"
+gem "elasticsearch", "~> 7.4.0"
```

Comment:
Why did you change this line? dedot_filter seems not to require this version. https://github.com/lunardial/fluent-plugin-dedot_filter

Besides, No need to change since `~> 7.0` installs v7.4.0. see https://guides.rubygems.org/patterns/

Reply:
dedot plugin was not installed before this patch, the version 1.0 is working with fluent >=0.14

I will correct the deployed version of elasticsearch gem